tests/integration/standard/test_metadata.py

```diff
@@ -1103,12 +1103,13 @@ def test_legacy_tables(self):
 
 CREATE TABLE legacy.composite_comp_no_col (
     key blob,
-    column1 'org.apache.cassandra.db.marshal.DynamicCompositeType(org.apache.cassandra.db.marshal.TimeUUIDType, org.apache.cassandra.db.marshal.BytesType, org.apache.cassandra.db.marshal.UTF8Type)',
```

Comment:
I don't want to beat this to death since this structure is not expressible in CQL in this version, but aren't we losing something by not showing DCT is involved?
If this seems fine to the C* stakeholders, I'm fine with it too.